docs/source/noise/row_noise.rst

```diff
@@ -89,3 +89,48 @@ parameter:
 
 When applying :code:`omit_row` noise, each row of data is selected for omission
 independently with probability :code:`row_probability`.
+
+Duplicate with guardian
+-----------------------
+
+A known challenge in entity resolution is people being reported multiple
+times at different addresses. This can occur when family structures are
+complex and children spend time at multiple households. A related
+challenge occurs with college students, who are sometimes counted both at their
+university and at their guardian’s home address.
+
+To facilitate this type of error, we have simulants assigned to guardians
+within the simulation. Sometimes, those guardians may live at
+different addresses than their dependents. In this case, there is an
+opportunity for duplication. Since this mechanism occurs within the
+simulation, there is a natural maximum on how many simulants can be duplicated
+in this way.
+
+Guardian-based duplication is applied to two mutually exclusive categories of
+simulants based on age and GQ status: Simulants younger than 18 (<18) and not
+in GQ and simulants under 24 (<24) and in college GQ.
+
+For each of the two categories of simulants, the maximum duplication rate will
+be calculated based on those who have a guardian living at a different address
+in the sim. Most simulants in college GQ will have a guardian at a
```

Comment:
```suggestion
For each of the two categories of simulants, a maximum duplication rate will
be calculated based on those who have a guardian living at a different address.
Most simulants in college GQ will have a guardian at a
```
Changing "the" to "a" since, if we delete the above paragraph, we won't have already mentioned the maximum. Also, I don't think it's necessary to mention "the sim" here.